gio/src/action_map.rs

```diff
@@ -0,0 +1,50 @@
+// Take a look at the license at the top of the repository in the LICENSE file.
+
+use crate::{prelude::*, ActionEntry, ActionMap, SimpleAction};
+use glib::{clone, Cast, IsA};
+
+pub trait ActionMapExtManual {
+    #[doc(alias = "g_action_map_add_action_entries")]
+    fn add_action_entries(&self, entries: Vec<ActionEntry<Self>>) -> Result<(), glib::BoolError>
+    where
+        Self: IsA<ActionMap>;
+}
+
+impl<O: IsA<ActionMap>> ActionMapExtManual for O {
+    fn add_action_entries(&self, entries: Vec<ActionEntry<Self>>) -> Result<(), glib::BoolError> {
```

Comment:
```suggestion
    fn add_action_entries(&self, entries: impl IntoIterator<Item = ActionEntry<Self>>) -> Result<(), glib::BoolError> {
```

maybe?

Reply:
Doesn't help much because you can't pass in a `&[some_action_entry]` for example

Reply:
Of course not, but you can pass in a `[some_action_entry]` :) `&[T]` `IntoIterator` gives you `&T`.

Reply:
Oh right, i didn't knew I could pass `[T]`.

Reply:
Since Rust 2022 this works nicely. I'm using that in many places in GStreamer now, makes things nicer quite often.